scapy/arch/windows/__init__.py

```diff
@@ -868,10 +870,10 @@ def _ask_user():
                         self.load_from_powershell()
                         return
                 _error_msg = "Could not start the pcap service ! "
-            warning(_error_msg +
-                    "You probably won't be able to send packets. "
-                    "Deactivating unneeded interfaces and restarting Scapy might help. "  # noqa: E501
-                    "Check your winpcap and powershell installation, and access rights.")  # noqa: E501
+            _error_msg += "You probably won't be able to send packets. "
```

Comment:
I don't understand this change. Wouldn't it be cleaner to keep the previous version and cut the longest lines to remove the `noqa`?

Reply:
noqa removed